### ISSUE TITLE
Synergies with CLLD and use of CLDF formats

### ISSUE THREAD

**Issue by PaulHeggarty** [lingdb/Sound-Comparisons#464]
See also:  https://github.com/lingdb/Sound-Comparisons/issues/456

- [ ] Create a new field in the `Transcriptions` table, named `CLPA Transcription`.  @LinguList should advise on whether the new field should initially be blank, or a copy of the entry in the current transcription field.
- [ ] Set up rights for @LinguList and @xrotwang to access  the `Transcriptions` table on the server.

**Comment by LinguList** [comment]
I don't think that is what we need. Jakob adjusted my tool for automatic segmentation two years ago, but it was never introduced. In the meantime, I guess it's easier if we deal with these things from lexibank, unless you want to show alignments at some points in your tool. For us it's enough to just have the transcriptions as they are right now.
